.changeset/pretty-mice-share.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
+
+Internal cleanup from errors in console regarding `act()`.
```

Comment:
Users wouldn't really care about that, but you changed some functionality in the component that people use. Maybe explain that instead?

Reply:
@LekoArts Customers are not affected by the other change either, i'm converting this to an empty changeset